WordPress/src/main/java/org/wordpress/android/ui/WPWebViewActivity.java

```diff
@@ -366,6 +366,10 @@ public static String getAuthenticationPostData(String authenticationUrl, String
      * @return URL of the login page.
      */
     public static String getSiteLoginUrl(SiteModel site) {
+        if (site.isJetpackConnected()) {
```

Comment:
This was added in c3ce617d77cb9e9c0c73d301843b6e9a9d4bef59 because I was running into authentication issues with Jetpack sites and `WPWebViewActivity`. So trying to open `Details` or `Support`, for example, would fail for Jetpack sites even though the URL redirects to a public page 🤷‍♂️

Reply:
Reminder that this still hasn't been addressed.

Reply:
Moved to `ThemeWebActivity` in 442432a2b125c4959957728c4f62832b56acae4a.